packages/run-protocol/src/vaultFactory/prioritizedVaults.js

```diff
@@ -10,8 +10,8 @@ import { toVaultKey } from './storeUtils.js';
 
 /**
  *
- * @param {Amount<NatValue>} debtAmount
- * @param {Amount<NatValue>} collateralAmount
+ * @param {Amount<'nat'>} debtAmount
+ * @param {Amount<'nat'>} collateralAmount
```

Comment:
```suggestion
 * @param {Amount<NAT>} debtAmount
 * @param {Amount<NAT>} collateralAmount
```

If there is not a straightforward way to arrange this, then I can't speak for @Chris-Hibbert and @erights in approving tihs.

Reply:
This can totally be done, by defining `const NAT = 'nat'` but since `NAT` is `'nat'` then what the user sees in an IDE or type error will still be `'nat'`.